tests/elements-regression/tests/elements-regression.test.js

```diff
@@ -40,7 +40,11 @@ test.describe( 'Elementor regression tests with templates for CORE', () => {
 			await editorPage.closeNavigatorIfOpen();
 			await editorPage.loadTemplate( filePath );
 			if ( 'video' === widgetType ) {
-				await editorPage.waitForVideoLoaded();
+				await editorPage.waitForIframeToLoaded( { iframeSelector: EditorSelectors.videoIframe, elementToWaitFor: EditorSelectors.playIcon } );
```

Comment:
Maybe this code should be in waitForIframeToBeLoaded? I'm not sure it should, but we repeat the same code for both the published and unpublished parts

Reply:
It's not the same. Video and google maps have different iframes. Also before publishing we have elementor-preview-iframe after publish it is missing. waitForIframeToLoaded is just reused 4 times. 
Maybe I can find a better solution

Reply:
fixed